### PR TITLE
Several updates

### PR DESCRIPTION
**Pull request by chadagreene**
Combining multiple commits in one PR.

### CODE REVIEW

**Review by alex-s-gardner — COMMENTED** [review]
Looks fantastic! see in-line comments.

File: Documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Using GEMB
+![](figures/DependencyGraph.pdf)
+The simplest example of how to use GEMB can be found in `MASTER_RUN.m`, which loads some local climate data, defines some constants, and enters all of that into the `GEMB` function. The dependency graph above shows that `GEMB` calls several other functions that each model a given process such as accumulation, densification, melt, shortwave radiation. etc.
+
+# Function List 
+**`accumulation`** adds precipitation and deposition to the model grid.
+
+**`albedo`** calculates snow, firn, and ice albedo.
+
+**`combineStrucData_GEMB`** combines GEMB input data into structure for passing to parfor loop.
```
[the remark this answers is not in the document]
Comment (chadagreene) [reply Documentation/README.md:10]: I was thinking the same thing, and spent the day laying the groundwork to move away from structures.

File: Documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* F: refreeze of column [kg m-2]
+* FAC: firn air content within column [m]
+* M: melt of column [kg m-2]
+* P: precipitation input [kg m-2]
+* R: runoff of column [kg m-2]
+* Ra: rain added to column [kg m-2]
+* S: structure of all model input
+* T: temperature of each layer [K]
+* Ta: 2m air temperature input [K]
+* W: liquid water content of each layer [kg m-2]
```
[the remark this answers is not in the document]
Comment (chadagreene) [reply Documentation/README.md:48]: The variable list is somewhat of a placeholder for now, so we should probably save the fine-tuning for later.

File: Documentation/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Using GEMB
+![](figures/DependencyGraph.pdf)
+The simplest example of how to use GEMB can be found in `MASTER_RUN.m`, which loads some local climate data, defines some constants, and enters all of that into the `GEMB` function. The dependency graph above shows that `GEMB` calls several other functions that each model a given process such as accumulation, densification, melt, shortwave radiation. etc.
+
+# Function List
```
[the remark this answers is not in the document]
Comment (chadagreene) [reply Documentation/README.md:5]: I've been thinking about this and looking at Julia style guides for when to use underscores versus when to use camel case. If GEMB gets rewritten in Julia, I'd personally like the MATLAB and Julia versions to look and feel as similar to each other as possible. But I also understand that any changes to the current MATLAB version could diverge from the ISSM version and make @NJSchlegel's life harder.

**Pull request merged**